src/graphene-rect.c

```diff
@@ -829,3 +829,36 @@ graphene_rect_zero (void)
 {
   return &_graphene_rect_zero;
 }
+
+/**
+ * graphene_rect_scale:
+ * @s_h: horizontal scale factor
+ * @s_v: vertical scale factor
+ * @res: (out caller-allocates): return location for the
+ *   scaled rectangle
+ *
+ * Scales the size and origin of a rectangle horizontaly by @s_h,
+ * and vertically by @s_v. The result @res is normalized.
+ *
+ * Returns: (transfer none): a scaled rectangle
+ *
+ * Since: 1.8.6
```

Comment:
This is going to be `Since: 1.10` once I branch master.

Reply:
I've added the commit bumping the version and adding the relevant macros. Let me know if that works for you.